### PR TITLE
feat():Users should be able to update and delete articles

### PR DESCRIPTION
**Pull request by kenneth051**
**What does this PR do?**
Enables users to delete and edit articles.

**Description of task to be completed**
Articles form the backbone of our application. This task enables users to delete and edit the contents of an article. Only authenticated users can delete and edit articles.

**How should this be manually tested?**
Clone the repository, checkout into this branch
- Run `yarn install` to install all dependencies
- Run  `yarn test` to run the tests
- Run `yarn start`
- login
         ```"email": "ddumba.kenneth11@andela.com"```
         ```"password": "Tteeeeee123"```
-  go to the create article endpoint  ```http://localhost:3000/createArticle/```
- Go to my articles endpoint, ```http://localhost:3000/myArticles``` click on edit or delete

**Relevant Pivotal Tracker Stories**

[#160583963](https://www.pivotaltracker.com/n/projects/2198584/stories/160583963)

#### Any background context you want to add?
N/A

**Screen shots**

screen shot 2018-11-19 at 15 29 29

screen shot 2018-11-20 at 14 30 41

screen shot 2018-11-20 at 18 59 06

### CODE REVIEW

**Review by balaakagordon — APPROVED** [review]
LGTM

**Comment by Edward-K1** [comment]
Good work @kenneth051. The update and delete work as expected, but please update the 'my articles' option on the navbar dropdown to point to a user's articles.

File: src/index.css [hunk 1]
```diff
@@ -122,6 +130,182 @@ input:focus, .btn:active {
...
+  color: chocolate;
+  font-size: 2em;
+}
+
+hr {
+  margin-bottom: 40px;
+  width: 90%;
+}
+
+.card {
```
Comment (Edward-K1) [inline src/index.css]: Please don't override the default bootstrap card class. This will make other features dependent on it display incorrectly.

**Comment by Edward-K1** [comment]
LGTM.

**Review by nakimera — CHANGES_REQUESTED** [review]
Would you change the colour of the article title on the article cards to blue and display the author's profile picture on the article cards.

**Review by j0flintking02 — APPROVED** [review]
LGTM

**Review by Edward-K1 — APPROVED** [review]
LGTM.

**Review by nakimera — APPROVED** [review]

**Pull request merged**